CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+#### 0.3.4
+  * Add excludeOverriden to QueryOptions which removes declarations that were
+    overriden by a base class.
```

Comment:
How about

#### 0.3.4
  * Add `excludeOverriden` to QueryOptions which removes declarations that were overriden within the class hierarchy.